README.md

```diff
@@ -1,35 +1,47 @@
 # visualize_packs
+Visualize_packs helps you visualize the structure, intended and actual, of your package-based Ruby application.
 
-This gem helps visualize relationships between packs.
+This gem takes a minimal approach in that it only outputs a graph in the format of [graphviz](https://graphviz.org/)' [dot language](https://graphviz.org/doc/info/lang.html). Install graphviz and use one of the chains of commands from below to generate full or partial images of the graphs of your application.
```

Comment:
This is pretty interesting... so it's not actually generating a PNG... it's just generating something that adheres to the dot language specification and therefore can be loaded bo the dot program?

Reply:
that's correct